be/src/common/logconfig.cpp

```diff
@@ -85,7 +85,8 @@ static void dump_trace_info() {
         // dump query_id and fragment id
         auto query_id = CurrentThread::current().query_id();
         auto fragment_instance_id = CurrentThread::current().fragment_instance_id();
-        char buffer[256] = {};
+        const std::string lake_filename = CurrentThread::current().get_lake_filename();
+        char buffer[512] = {};
```

Comment:
consider to use std::string. 

or use snprintf to avoid overflow.

Reply:
use `snprintf()` for url, because it's length is unfixed.